docs/pages/react/solana-wallets/signing-and-sending-solana.mdx

```diff
@@ -0,0 +1,277 @@
+sendTransaction({
+  transfer: {
+    // Amount in the atomic values of solana (Sol)
+    amount: 123,
+    toAddress: "<AddressToSendTo>",
+  },
+});
+```
+
+Using instructions:
+
+If you want finer grained control like transfer and nonceAdvance you can pass instructions by using the exposed [web3 SystemProgram](https://solana-foundation.github.io/solana-web3.js/classes/SystemProgram.html).
+
+```tsx
+sendTransaction({
+  instructions: [
+    SystemProgram.transfer({
+      fromPubkey: new PublicKey("<FromAddress>"),
+      toPubkey: new PublicKey("<ToAddress>"),
+      lamports: params.transfer.amount,
+    }),
+  ],
+});
+```
+
+# Sign messages with useSolanaSignMessage
+
+To sign a message, either a strin or a byte array, using your Solana wallet, simply use the `useSolanaSignMessage` hook.
+
+```tsx [sign-solana-message.tsx]
+import { useSolanaSignMessage } from "@account-kit/react";
+import { SolanaSigner } from "@account-kit/signer";
+
+function MyComponent() {
+  const {
+    signer,
+    data,
+    signMessage,
+    data: signedMessage,
+    reset,
+  } = useSolanaSignMessage({});
+
+  if (!signer) {
+    return <div>Loading signer...</div>;
+  }
+
+  return (
+    <div>
+      <button onClick={() => signMessage({ message: "Here is my message" })}>
+        {" "}
+        Sign{" "}
+      </button>
+      Message Signed: {signedMessage}
+    </div>
+  );
+}
+```
+
+```tsx
+import { useSolanaSignMessage } from "@account-kit/react";
+import { SolanaSigner } from "@account-kit/signer";
+
+function MyComponent() {
+  const {
+    signer,
+    data,
+    signMessage,
+    data: signedMessage,
+    reset,
+  } = useSolanaSignMessage({});
+
+  if (!signer) {
+    return <div>Loading signer...</div>;
+  }
+
+  return (
+    <div>
+      <button onClick={() => signMessage({ message: "Here is my message" })}>
+        {" "}
+        Sign{" "}
+      </button>
+      Message Signed: {signedMessage}
+    </div>
+  );
+}
+```
+
+# Multi-signature support using PreSign
+
+You can implement multi-signature behavior using the `PreSign` feature to authorize multiple signers on a transaction before execution.
+
+### Example use case: adding additional signers
+
+A common pattern involves supporting multiple signers for a single transaction. For example, in our demo environment, we generate a temporary (or "throwaway") account that can hold various tokens. However, this account alone isn't sufficient for transaction security or coordination.
+
+To enable multi-signature functionality, we attach additional signers to the transaction using `preSign`. This allows the transaction to be authorized by multiple parties before being broadcast, ensuring that:
+
+- Multiple stakeholders can approve a transaction.
+- Tokens can exist at a shared account address with controlled access.
+- Coordination between signers happens off-chain via pre-signed approvals.
+
+This pattern is useful for collaborative workflows, DAO interactions, or shared treasury use cases.
```

Comment:
The documentation seems good so far, but there's one line that seems off.

After looking into multi-signer wallets and other resources, I believe most of the integration into those wallets creates a PDA.

This PDA then allows the transaction to send for a multi-signer wallet, which is a program.

And I don't think it uses multiple signers in that case.

Here's a link to the relevant discussion: https://alchemyinsights.slack.com/archives/C082ZDA1822/p1747079038669899?thread_ts=1747075000.578589&cid=C082ZDA1822